tests/gtest/avif_fuzztest_helpers.h

```diff
@@ -188,38 +206,34 @@ ImagePtr AddGainMapToImage(
 
 inline auto ArbitraryAvifImageWithGainMap() {
   return fuzztest::Map(
-      AddGainMapToImage, ArbitraryAvifImage(), ArbitraryAvifImage(),
-      fuzztest::Arbitrary<int32_t>(), fuzztest::Arbitrary<int32_t>(),
-      fuzztest::Arbitrary<int32_t>(),
-      fuzztest::InRange<uint32_t>(1, std::numeric_limits<uint32_t>::max()),
-      fuzztest::InRange<uint32_t>(1, std::numeric_limits<uint32_t>::max()),
-      fuzztest::InRange<uint32_t>(1, std::numeric_limits<uint32_t>::max()),
-      fuzztest::Arbitrary<int32_t>(), fuzztest::Arbitrary<int32_t>(),
-      fuzztest::Arbitrary<int32_t>(),
-      fuzztest::InRange<uint32_t>(1, std::numeric_limits<uint32_t>::max()),
-      fuzztest::InRange<uint32_t>(1, std::numeric_limits<uint32_t>::max()),
-      fuzztest::InRange<uint32_t>(1, std::numeric_limits<uint32_t>::max()),
-      fuzztest::InRange<uint32_t>(1, std::numeric_limits<uint32_t>::max()),
-      fuzztest::InRange<uint32_t>(1, std::numeric_limits<uint32_t>::max()),
-      fuzztest::InRange<uint32_t>(1, std::numeric_limits<uint32_t>::max()),
-      fuzztest::InRange<uint32_t>(1, std::numeric_limits<uint32_t>::max()),
-      fuzztest::InRange<uint32_t>(1, std::numeric_limits<uint32_t>::max()),
-      fuzztest::InRange<uint32_t>(1, std::numeric_limits<uint32_t>::max()),
-      fuzztest::Arbitrary<int32_t>(), fuzztest::Arbitrary<int32_t>(),
-      fuzztest::Arbitrary<int32_t>(),
-      fuzztest::InRange<uint32_t>(1, std::numeric_limits<uint32_t>::max()),
-      fuzztest::InRange<uint32_t>(1, std::numeric_limits<uint32_t>::max()),
-      fuzztest::InRange<uint32_t>(1, std::numeric_limits<uint32_t>::max()),
-      fuzztest::Arbitrary<int32_t>(), fuzztest::Arbitrary<int32_t>(),
-      fuzztest::Arbitrary<int32_t>(),
-      fuzztest::InRange<uint32_t>(1, std::numeric_limits<uint32_t>::max()),
-      fuzztest::InRange<uint32_t>(1, std::numeric_limits<uint32_t>::max()),
-      fuzztest::InRange<uint32_t>(1, std::numeric_limits<uint32_t>::max()),
-      fuzztest::Arbitrary<uint32_t>(),
-      fuzztest::InRange<uint32_t>(1, std::numeric_limits<uint32_t>::max()),
-      fuzztest::Arbitrary<uint32_t>(),
-      fuzztest::InRange<uint32_t>(1, std::numeric_limits<uint32_t>::max()),
-      fuzztest::Arbitrary<bool>());
+      AddGainMapToImage, ArbitraryAvifImage(),
+      /*gain_map=*/ArbitraryAvifImage(),
+      /*gain_map_min_max0=*/ArbitraryMinMaxSignedFraction(),
+      /*gain_map_min_max1=*/ArbitraryMinMaxSignedFraction(),
+      /*gain_map_min_max2=*/ArbitraryMinMaxSignedFraction(),
+      /*gain_map_gamma_n0=*/fuzztest::NonZero<uint32_t>(),
+      /*gain_map_gamma_n1=*/fuzztest::NonZero<uint32_t>(),
+      /*gain_map_gamma_n2=*/fuzztest::NonZero<uint32_t>(),
```

Comment:
Should these three numerators use `Arbitrary` instead of `NonZero`?

Reply:
No this is working as intended. Gamma must be >0.